### PR TITLE
[MISC] Fix typos in comments

### PR DESCRIPTION
**Pull request by nfxpnk**
## Purpose

Fixing typos in code comments enhances code readability and comprehension for developers and collaborators.

JIRA issue: No Jira access and no issue

## Proposed changes

Fix misc typos in comments

### CODE REVIEW

File: win32ss/user/user32/controls/ghost.c [hunk 1]
```diff
@@ -331,7 +331,7 @@ Ghost_OnNCPaint(HWND hwnd, HRGN hrgn, BOOL bUnicode)
 {
     HDC hdc;
 
-    // do the default behaivour
+    // do the default behavior
```
Comment (HBelusca) [inline win32ss/user/user32/controls/ghost.c:334]:
```suggestion
    // do the default behaviour
```

File: win32ss/reactx/dxg/historic.c [hunk 2]
```diff
@@ -18,7 +18,7 @@
 * The function DxDxgGenericThunk redirects DirectX calls to other functions.
 *
 * @param ULONG_PTR ulIndex
-* The functions we want redirct
+* The functions we want redirect
```
Comment (HBelusca) [inline win32ss/reactx/dxg/historic.c:21]:
```suggestion
* The functions we want to redirect
```

File: sdk/tools/mkisofs/schilytools/mkisofs/write.c [hunk 3]
```diff
@@ -3001,7 +3001,7 @@ insert_padding_file(size)
 		dw_tail = dwpnt;
 	}
 
-	/* aloocate memory as a "Table" file */
+	/* allocate memory as a "Table" file */
```
Comment (HBelusca) [inline sdk/tools/mkisofs/schilytools/mkisofs/write.c:3004]: 3rd-party code, don't change.

File: dll/opengl/glu32/src/libnurbs/interface/insurfeval.cc [hunk 4]
```diff
@@ -1571,7 +1571,7 @@ void OpenGLSurfaceEvaluator::inMap2fEM(int which, int k,
 
   REAL *data = temp_em->ctlPoints;
   
-  temp_em->uprime = -1;//initilized
+  temp_em->uprime = -1;//initialized
```
Comment (HBelusca) [inline dll/opengl/glu32/src/libnurbs/interface/insurfeval.cc:1574]: 3rd-party code, don't change.

File: win32ss/user/user32/controls/ghost.c (hunk 1 above)
Comment (binarymaster) [reply win32ss/user/user32/controls/ghost.c]: heh I see en-US vs. en-GB here (both are correct)

File: win32ss/user/user32/misc/ddeclient.c [hunk 5]
```diff
@@ -226,7 +226,7 @@ HCONV WINAPI DdeReconnect(HCONV hConv)
 	 * 1/ pConv is the conversation attached to the client window (it wouldn't be
 	 *    if a call to DdeReconnect would have already been done...)
 	 *    FIXME: is this really an error ???
-	 * 2/ the pConv conversation had really been deconnected
+	 * 2/ the pConv conversation had really been disconnected
```
Comment (binarymaster) [inline win32ss/user/user32/misc/ddeclient.c:229]: 3rd party code: https://github.com/reactos/reactos/blob/56d95154eeaf2e96d5f3c6cd47baa03e50435235/media/doc/WINESYNC.txt#L338C46-L338C74

**Review by JoachimHenze — APPROVED** [review]
Approved. But we have to make sure, that we will not touch any 3rd party code.
@binarymaster can you check whether any of that code is 3rd party?

**Review by binarymaster — CHANGES_REQUESTED** [review]
Hello,
Thanks for your interest in ReactOS!
Since you have modified the files containing code, we ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
[Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.

To change this for future PR's, you can update it for globally (for the system):
```
git config --global user.name "Your Name"
git config --global user.email you@example.com
```

Or just for the current repository:
```
git config user.name "Your Name"
git config user.email you@example.com
```

Also please set up your GitHub account accordingly, so maintainers can use Squash & Merge button without getting the author field modified back to nickname by GitHub - set your full name in [GitHub profile settings](https://github.com/settings/profile) (it should match your `user.name` setting).

Regards,

**Review by binarymaster — APPROVED** [review]

**Review by learn-more — APPROVED** [review]

**Pull request merged**